android/BOINC/app/src/main/java/edu/berkeley/boinc/adapter/PrefsListAdapter.java

```diff
@@ -103,10 +103,10 @@ public View getView(int position, View convertView, ViewGroup parent) {
 	    					value = NumberFormat.getInstance().format(item.status) + this.activity.getString(R.string.prefs_unit_celsius);
 	    					break;
 	    				case MEGABYTES:
-	    					value = Formatter.formatShortFileSize(this.activity, (long)(item.status.doubleValue() * 0x100000));
+	    					value = Formatter.formatShortFileSize(this.activity, (long)(item.status * 0x100000));
```

Comment:
@adamradocz, did you verify that this logic still produces valid numbers?

Reply:
`item.status` returns `double` value by default.